app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1099,6 +1099,49 @@ class PatientRepositoryAndroidTest {
         .dispose()
   }
 
+  @Test
+  fun verify_deleted_PrescribedDrugs_are_not_included_when_fetching_recent_patients() {
+    val facilityUuid = testData.qaUserFacilityUuid()
+    val recentPatient1 = savePatientWithPrescribedDrug(facilityUuid = facilityUuid)
+    savePatientWithPrescribedDrug(facilityUuid = facilityUuid, deletedAt = Instant.now())
```

Comment:
Can you assign a variable to this as well?

`recentPatient2 = savePatientWithPrescribedDrug(facilityUuid = facilityUuid, deletedAt = Instant.now())`

Reply:
done